covid_xprize/examples/predictors/lstm/data/Additional_Context_Data_Global.csv

```diff
@@ -8,12 +8,17 @@ Argentina,ARG,45195774,16.2585101,91.87,11.11778888,11683.94962,28.3,1.176868831
 Aruba,ABW,106766,,,,,,,,,,,
 Australia,AUS,25499884,3.249129487,86.012,15.65647523,57373.68668,29,3.042002085,12.62,9.071138937,2.86,15.38876602,665384
 Austria,AUT,9006398,107.2069267,58.297,19.00156646,51461.95478,20.1,2.228314728,14.13,4.587225585,4.29,6.86986793,130260
+Azerbaijan,AZE,10127874,,,,,,,,,,,
```

Comment:
I'm not in a position to validate any of the numbers, but I would like to see an attribution file, similar to the one [here](https://github.com/leaf-ai/esp-demo/blob/master/Team2-Covid19/data/input/data_sources.txt) That way we'll have a record of where the data came from.

Reply:
Done